packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -81,12 +58,13 @@ export default {
         },
         maxStarRating: {
             type: Number,
-            default: 5
+            default: 5,
+            validator: value => value > 0
```

Comment:
I think at the moment it would also allow something like 2.5? Happy for you to address in a future PR though

Reply:
Nice spot! Fixed.